log-viewer-cli/pom.xml

```diff
@@ -93,7 +87,6 @@
             </plugin>
         </plugins>
     </build>
-
```

Comment:
The empty line between sections (<build> and <dependencies>) looks good. Please, return.

Reply:
no problem!